### ISSUE TITLE
[simprod] includes projects which are not in combo (Trac #1743)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1735]
This causes warnings in sphinx

```text
/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:23: WARNING: autodoc: failed to import module u'icecube.simprod.util.corsika_binary_stager'; the following exception was raised:
Traceback (most recent call last):
  File "/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py", line 385, in import_object
  File "/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/corsika_binary_stager.py", line 13, in <module>
    class CorsikaBinaryStager(CorsikaBinary):
NameError: name 'CorsikaBinary' is not defined
/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:47: WARNING: autodoc: failed to import module u'icecube.simprod.util.gaussSpreadDOMeff'; the following exception was raised:
Traceback (most recent call last):
  File "/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py", line 385, in import_object
  File "/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/gaussSpreadDOMeff.py", line 6, in <module>
    from iceprod.modules import ipmodule
ImportError: No module named iceprod.modules
/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:55: WARNING: autodoc: failed to import module u'icecube.simprod.util.modifyevent'; the following exception was raised:
Traceback (most recent call last):
  File "/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py", line 385, in import_object
  File "/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/modifyevent.py", line 14, in <module>
    from iceprod.modules import ipmodule
ImportError: No module named iceprod.modules
/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:63: WARNING: autodoc: failed to import module u'icecube.simprod.util.splitter'; the following exception was raised:
Traceback (most recent call last):
  File "/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py", line 385, in import_object
  File "/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/splitter.py", line 5, in <module>
    from iceprod.modules import ipmodule
ImportError: No module named iceprod.modules
```

Migrated from https://code.icecube.wisc.edu/ticket/1743
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "This causes warnings in sphinx\n{{{\n/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:23: WARNING: autodoc: failed to import module u'icecube.simprod.util.corsika_binary_stager'; the following exception was raised:\nTraceback (most recent call last):\n  File \"/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py\", line 385, in import_object\n  File \"/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/corsika_binary_stager.py\", line 13, in <module>\n    class CorsikaBinaryStager(CorsikaBinary):\nNameError: name 'CorsikaBinary' is not defined\n/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:47: WARNING: autodoc: failed to import module u'icecube.simprod.util.gaussSpreadDOMeff'; the following exception was raised:\nTraceback (most recent call last):\n  File \"/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py\", line 385, in import_object\n  File \"/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/gaussSpreadDOMeff.py\", line 6, in <module>\n    from iceprod.modules import ipmodule\nImportError: No module named iceprod.modules\n/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:55: WARNING: autodoc: failed to import module u'icecube.simprod.util.modifyevent'; the following exception was raised:\nTraceback (most recent call last):\n  File \"/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py\", line 385, in import_object\n  File \"/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/modifyevent.py\", line 14, in <module>\n    from iceprod.modules import ipmodule\nImportError: No module named iceprod.modules\n/Users/kmeagher/icecube/combo/release/sphinx_build/source/python/icecube.simprod.util.rst:63: WARNING: autodoc: failed to import module u'icecube.simprod.util.splitter'; the following exception was raised:\nTraceback (most recent call last):\n  File \"/private/var/folders/rc/g_4_lyp9039cj1586zzg88f40000gn/T/pip-build-A327aa/sphinx/sphinx/ext/autodoc.py\", line 385, in import_object\n  File \"/Users/kmeagher/icecube/combo/release/lib/icecube/simprod/util/splitter.py\", line 5, in <module>\n    from iceprod.modules import ipmodule\nImportError: No module named iceprod.modules\n}}}", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "combo simulation", 
    "summary": "[simprod] includes projects which are not in combo", 
    "priority": "minor", 
    "keywords": "documentation", 
    "time": "2016-06-10T14:16:51", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160610T14:45:12: 
* olivas changed owner from "" to "david.schultz"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160615T18:38:23: olivas changed component from "cmake" to "simulation"

**Comment by icecube-trac** [comment]
Trac update at 20160728T16:45:29: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 148848:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148848"
> fix up some imports. closes #1735
> ```


* david.schultz changed status from "assigned" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas changed milestone from "August 2016" to ""
* olivas commented:

> Milestone August 2016 deleted
